nupic/algorithms/anomaly.py

```diff
@@ -84,22 +93,15 @@ def __init__(self, slidingWindowSize=None, mode=MODE_PURE, binaryAnomalyThreshol
               models probability of receiving this value and anomalyScore
           - "weighted" - "pure" anomaly weighted by "likelihood"
               (anomaly * likelihood)
+          - OR type function and will be used directly to compute score
```

Comment:
"OR **callable** that will be used directly to compute score". Also, document the inputs and outputs of this callable that would enable one to use this without having the plough through the code to figure it out.